app/mailers/signup_confirmation_mailer.rb

```diff
@@ -6,7 +6,7 @@ def instructions(pre_auth_state:)
 
     res = mail to: "#{pre_auth_state.contact_info_value}",
          subject: @show_pin ?
-                    "Use PIN #{pre_auth_state.confirmation_pin} to confirm your email address" :
+                    "Please confirm your OpenStax account" :
```

Comment:
Why take the code out of the subject? I like it when confirmation codes are in the subject of emails or the first X characters of a text message so I can just read it directly.